core/templates/components/question-directives/modal-templates/question-editor-save-modal.component.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Component for question editor save modal.
+ */
+
+import { Component, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { AppConstants } from 'app.constants';
+import { ConfirmOrCancelModal } from 'components/common-layout-directives/common-elements/confirm-or-cancel-modal.component';
+
+@Component({
+  selector: 'oppia-question-editor-save-modal',
+  templateUrl: './question-editor-save-modal.component.html'
+})
+export class QuestionEditorSaveModalComponent extends ConfirmOrCancelModal
+  implements OnInit {
+  MAX_COMMIT_MESSAGE_LENGTH: number;
```

Comment:
Assigning value here will also work.

Reply:
Done.

Reply:
Its creating problem while saving the question